sample/build.gradle

```diff
@@ -45,4 +45,5 @@ dependencies {
   }
   compile 'com.android.support:appcompat-v7:23.2.1'
   compile 'com.android.support:design:23.2.1'
+  compile 'com.mapzen.tangram:tangram:0.2-SNAPSHOT'
```

Comment:
I re-added this b/c I was running into issues with IntelliJ/Android Studio not resolving Tangram classes in the sample app. Command line build would work fine but the IDE was full of red. Is it working normally for you?